package.json

```diff
@@ -6,15 +6,17 @@
   "module": "./lib/main/ts/api/Main.js",
   "scripts": {
     "prepublishOnly": "tsc",
-    "build": "tsc && tslint --project .",
-    "test": "yarn build && mocha -r esm lib/test/**/*Test.js"
+    "lint": "tslint --project .",
+    "build": "tsc",
+    "test": "yarn build && yarn lint && mocha -r esm --require ts-node/register --require source-map-support/register --recursive --reporter mocha-multi-reporters --reporter-options configFile=mocha-multi-reporters-config.json src/test/ts/**/*Test.ts"
```

Comment:
That's one hell of a command :joy:

Reply:
Can we pull the CLI arguments out into a "mocha" field in the package.json?

Reply:
I've pulled it out into a `.mocharc.yml` file. Is that ok?

Reply:
Much nicer thanks!